src/ReplGPT.jl

```diff
@@ -4,13 +4,62 @@ import OpenAI
 import ReplMaker
 import Markdown
 
+@static if VERSION >= v"1.6"
+    using Preferences
+end
+
 const apiKeyName = "OPENAI_API_KEY"
+const apiPrefName = "openai_api_key"
 
-function call_chatgpt(s)
+"""
+    function getAPIkey()
+
+Returns an OpenAI API key to use from either the `LocalPreferences.toml` file or the
+`OPENAI_API_KEY` environment variable. If neither is present, returns `missing`.
+"""
+function getAPIkey()
+    key = missing
+
+    # try to load key from Preferences:
+    @static if VERSION >= v"1.6"
+        key = @load_preference(apiPrefName, missing)
+    end
```

Comment:
Check for Julia version can also be dropped here

Reply:
Done!